pep/app/services/current-user.ts

```diff
@@ -171,7 +171,7 @@ export default class CurrentUserService extends Service {
         }
 
         // if the user is logged in, apply the new prefs locally, then save the user
-        if (this.session.isAuthenticated && this.user) {
+        if (this.session.isAuthenticated && this.user && this.user.userType !== 'Group') {
```

Comment:
i think we should make this at least a unioned type, `type UserType = 'Group' | 'Individual'`  in the user model instead of comparing on the raw string directly.